beamlines/dev_env/configuration/meta.yaml

```diff
@@ -0,0 +1,13 @@
+package:
+  name: nslsii_dev_configuration
+  version: '1'
+
+build:
+  number: 1
+
+requirements:
+  run:
+    - conda_etc
+
+about:
+  summary: 'Development Enviroment server configuration'
```

Comment:
"Development environment configurations to point metadatastore and filestore at localhost"